src/components/FilterTodos/FilterTodos.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import classNames from 'classnames';
+import { FilterType } from '../../types/FilterType';
+
+type Props = {
+  todosCount: [number, number];
+  filterType: FilterType;
+  onFilter: (filterType: FilterType) => void;
+};
+
+export const FilterTodos: React.FC<Props> = ({
+  todosCount, onFilter, filterType,
+}) => {
+  const [numOfActive, numOfCompleted] = todosCount;
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {filterType === FilterType.COMPLETED
```

Comment:
Display the number of items left instead of always showing '0 items left' when filterType is FilterType.COMPLETED

Reply:
It was number of items left, but @mykhalenych, asked to change it. Look up

Reply:
check my comment

Reply:
what do you think about it?

Reply:
This is example [https://prnt.sc/m7Tk3vxD9N63](url).
I had the same. But @mykhalenych asked to change it, like in this video:
[https://github.com/mate-academy/react_todo-app-loading-todos/pull/471#pullrequestreview-1410627015](url).
So when you show only completed tasks, there are always 0 tasks left to do.